lab-yunjoo/server.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
```

Comment:
You've learned a LOT since we last talked - You did a good job of creating this server, and it looks like you have a good understanding of how to create server responses.